packages/backend/src/models/whiteboard.ts

```diff
@@ -91,7 +91,7 @@ export type Operation =
   | {
     type: OperationType.NOTE_MOVE;
     data: {
-      change: Pick<Note, 'id' | 'position>;
+      change: Partial<Note> & Pick<Note, 'id'>;
```

Comment:
I understood your comment that this should be e.g. `Partial<Note> & Pick<Note, 'id' | 'position'>` @wgslr 

However, due to an " `Coordinates | undefined` not assignable to `Coordinates` "-error in `whiteboard-controller.ts`  I reverted to this.

Reply:
That's because our protobuf library, ts-proto, likes to generate all fields with type `TypeOfField | undefined`. I'm pretty sure that's not compliant with protobuf specification, but rather their workaround for the time that protobuf3 didn't have `optional` modifier for fields.
And thus typescript indicates the possibility that incoming message will not contain the `position` field. This possibility has to be ruled out sooner or later. In your code, it is done in the `handleOperation` func. I'd argue a better place for that is the `whiteboard-controller`, because it's job is to clear any quirks of the communication layer and pass a clearly defined Operation further. 
So I did that, because I'm stubborn, along with cleaning up NOTE_UPDATE which doesn't need to check for `position` field in [1c1e327](https://github.com/wgslr/trunkless-whiteboard/pull/19/commits/1c1e327c9b22d07cd34ca85ee611bbdca11844ba).

Reply:
Good that you got the NOTE_UPDATE and NOTE_MOVE operations cleaned up like this👍